work_lights.py

```diff
@@ -177,13 +190,8 @@ def main():
   while now_time < WORK_DAY_END:
     
     # Identify if there is a GVC coming up or not
-    event_type = GetCalendarEvents(service)
-  
-    if event_type == 'GVC':
-      SetGVCColor(lights)
-
-    elif event_type == 'Ambient':
-      SetAmbientColor(lights)
```

Comment:
Instead of this design, how about having 3 methods
SetGVCColor()
SetAmbientSameColor()
SetAmbientMultiColor()

and in here, if event_type is Ambient
if MULTICOLOR then SetAmbientMultiColor()
else SetAmbientSameColor()

?